doc/accessibility-supported.md

```diff
@@ -0,0 +1,34 @@
+# How we make decisions on rules
+
+In order to adhere to the manifesto and at the same time be useful to developers and content creators, we evaluate all rules according to strict normative WCAG 2 interpretation. We also pay very strong attention to the normative portion of WCAG 2 known as [accessibility supported](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef).
+
+## Accessibility supported
+
+Boiled-down, accessibility supported means that in order for a technique to be valid, it must work on all viable platforms for all assitive technology that are widely used and freely available (paraphrased).
+
+We currently test the following AT combinations for support
+
+1. VoiceOver and Safari on OS X
+1. VoiceOver and Safari on iOS
+1. JAWS and IE on Windows
+1. NVDA and Firefox on Windows
+1. Talkback and Chrome on Android
+1. Dragon and Firefox on Windows
```

Comment:
We need some devices to actually support this policy. I personally don't have an Android device or Dragon.

Reply:
I don't think we've been very consistent about testing with Talkback, VO on iOS or Dragon. @dylanb Are you proposing we change this, or is this more of a nice to have list?

Reply:
This is the list we should be testing...

@marcysutton submit a request to get what you need to do your testing...also - Matt I. has access to devices in the Ann Arbor office and the teama11y mailing list is a resource you can also use